storage/postgresql/save.go

```diff
@@ -74,16 +74,9 @@ func deleteBeforeSaveSql(evt *nostr.Event) (string, []any, bool) {
 	return query, params, shouldDelete
 }
 
-func saveEventSql(evt *nostr.Event) (string, []any, error) {
-	const query = `INSERT INTO event (
-	id, pubkey, created_at, kind, tags, content, sig)
-	VALUES ($1, $2, $3, $4, $5, $6, $7)
-	ON CONFLICT (id) DO NOTHING`
-
-	var (
-		tagsj, _ = json.Marshal(evt.Tags)
-		params   = []any{evt.ID, evt.PubKey, evt.CreatedAt, evt.Kind, tagsj, evt.Content, evt.Sig}
-	)
-
-	return query, params, nil
+func saveEventSql(evt *nostr.Event) (string, []any) {
+	const query = `INSERT INTO event (id, pubkey, created_at, kind, tags, content, sig) VALUES ($1, $2, $3, $4, $5, $6, $7) ON CONFLICT (id) DO NOTHING`
```

Comment:
I don't see the value in removing the error return, or changing these newlines and groupings. The error is there because we're currently swallowing and might want to not do that.